### DISCUSSION TITLE
Way to get live testing leaderboard

### DISCUSSION THREAD

**Discussion by siddharth-tewari** [theOehrly/Fast-F1#104] [General]
Hello, is there a way to get leaderboard data from <a href="https://www.autosport.com/f1/live-text/f1-barcelona-pre-season-test-day-1/1112940/">https://www.autosport.com/f1/live-text/f1-barcelona-pre-season-test-day-1/1112940/</a>
to a .txt using fast-f1? I need to use it for some OBS project.
Sorry in advance if its not possible

**Comment by theOehrly** [comment]
No, that's nothing that FastF1 can do.
But you should be able to get this data from the html of the page with something like https://beautiful-soup-4.readthedocs.io/en/latest/

**Comment by siddharth-tewari** [comment]
Oh ok, will try as I don't know much about it.
